examples/OECC/README.md

```diff
@@ -0,0 +1,119 @@
+# OpenEdge Command Center #
+
+### Requirements 
+* Ubuntu Linux 22.04
+* OpenEdge media:
+    * PROGRESS_OE_12.8.3_LNX_64.tar.gz
+    * PROGRESS_OECC_SERVER_1.3.0_LNX_64.tar.gz
+    * PROGRESS_OECC_AGENT_1.3.0_LNX_64.bin
+* Reponse file for OpenEdge installation:
+    * response_12.8.3.ini
+    * Products:
+        * OE Adv. Ent. RDBMS
+        * 4GL Development System
+        * Progress ProdAS for OE
+
+**Notes:**
+* You can use the OpenEdge-Samples/examples/OECC/files/response_12.8.3.ini file as a template and specify the control codes in the file.
+
+### Components 
+* Ubuntu Virtual Machine
+    * OpenEdge Database
+    * PAS for OpenEdge
+    * OpenEdge Command Center agent
+    * Docker
+        * OpenEdge Command Center server
+        * MongoDB
+        * Prometheus
+        * Grafana
+
+![Components](./diagram.png)
+
+### Steps
+1. Create /files directory used for the installation files.
+~~~
+sudo mkdir -p /files
+sudo chown $USER /files
+~~~
+
+2. Copy the OpenEdge media files and the response file to /files.
+
+3. Clone the OpenEdge-Samples repo:
```

Comment:
Should this step come first?

Reply:
The cloning of the OpenEdge-Samples repo could come first.
However, the organization of the steps is based on grouping related steps:
- Step 1 and 2 go together.
- Step 3, 4, and 5 also go together.

Reply:
If these are logically grouped, then perhaps call out the intended process and number them per process.
eg.
Prepare the OpenEdge installation prerequisites:
1. ...
2. ...
Perform the installation of OpenEdge to Docker:
1. ...
2. ...

Reply:
Grouped steps.

Should each group have its own number?
Please take a look.

Reply:
It seems to make more sense that each section be it's own set of instructions so that it's easier to pick up after completion of a task.

Reply:
Updated the numbers in each group.